cmd/main.go

```diff
@@ -116,6 +116,7 @@ func main() {
 	rootCmd.AddCommand(
```

Comment:
maybe conditionally add based on the project version, and then override help here to say "some v1-specific commands not shown"